Clients/src/presentation/pages/Vendors/index.tsx

```
@@ -465,6 +474,76 @@ const Vendors = () => {
   }, [vendors, searchQuery, statusFilter]);
 
+  // Define how to get the group key for each vendor
+  const getVendorGroupKey = (vendor: VendorModel, field: string): string | string[] => {
+    switch (field) {
+      case 'review_status':
+        const statusMap: Record<ReviewStatus, string> = {
+          [ReviewStatus.NotStarted]: 'Not started',
+          [ReviewStatus.InReview]: 'In review',
+          [ReviewStatus.Reviewed]: 'Reviewed',
+          [ReviewStatus.RequiresFollowUp]: 'Requires follow-up',
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)